ansys/mapdl/core/_commands/preproc/meshing.py

```diff
@@ -2633,6 +2639,12 @@ def type(self, itype="", **kwargs):
     logical element type has been assigned via TYPE or XATT,,,TYPE.  For
     more information, see the discussion on setting element attributes in
     Meshing Your Solid Model in the Modeling and Meshing Guide.
+
+    Examples
+    --------
+
+    Set the type pointer to 2
+    >>> mapdl.type(2)
```

Comment:
```suggestion
    Set the type pointer to 2

    >>> mapdl.type(2)
```

See note in previous comment.